### DISCUSSION TITLE
Smart response not send

### DISCUSSION THREAD

**Discussion by martijnvanvloten** [dannagle/PacketSender#344] [Q&A]
Hi all,

Perhaps you might be able to help me. I use smart respnoses to simulate some hardware that I don't have at the moment. The issue that I'm having is that some smart responses are being sent, but other don't .
I copy/paste the exact string from the log to the smart respones, and the response is never send.. Any suggestions? I also tried to do it via hex, same story.
Response 1 and 2 are working normal, number 3 is not.

![image](https://github.com/dannagle/PacketSender/assets/74464567/d7bc5163-fb82-4e31-9cc9-5dace2d3d823)

Thanks,
Regards,
Martijn

**Comment by dannagle** [comment]
I think this feature request is the real answer to your problem: #332 .
